configserver/src/main/java/com/yahoo/vespa/config/server/http/LogRetriever.java

```diff
@@ -16,11 +20,20 @@ public class LogRetriever {
 
     private final CloseableHttpClient httpClient = VespaHttpClientBuilder.create().build();
 
-    public HttpResponse getLogs(String logServerUri) {
+    public HttpResponse getLogs(String logServerUri, Optional<Instant> deployTime) {
         HttpGet get = new HttpGet(logServerUri);
         try {
             return new ProxyResponse(httpClient.execute(get));
         } catch (IOException e) {
+            // It takes some time before nodes are up after first-time deployment, return empty log for up to 1 minute
+            // if getting logs fail
+            if (deployTime.isPresent() && Instant.now().isAfter(deployTime.get().minus(Duration.ofMinutes(1))))
```

Comment:
Won't this always be true if deployTime is present?
```suggestion
            if (deployTime.isPresent() && Instant.now().isBefore(deployTime.get().plus(Duration.ofMinutes(1))))
```

Reply:
You are right, thanks!